docs/reference/stream/zsink.md

```diff
@@ -295,9 +295,9 @@ val stream: ZIO[Any, Nothing, Int] =
 // Output: 10
 ```
 
-### From Effect
+### From ZIO
 
-The `ZSink.fromEffect` creates a single-value sink produced from an effect:
+The `ZSink.fromZIO` creates a single-value sink produced from an effect:
```

Comment:
I wonder whether "effect" (used end of line here) is still in common use in the doc after these methods renames.

Reply:
I might use "ZIO workflow" here.

Reply:
I'll apply the suggestion.